saleor/payment/utils.py

```diff
@@ -365,3 +368,21 @@ def is_currency_supported(currency: str, gateway_id: str, manager: "PluginsManag
     """Return true if the given gateway supports given currency."""
     available_gateways = manager.list_payment_gateways(currency=currency)
     return any([gateway.id == gateway_id for gateway in available_gateways])
+
+
+def get_last_payment(payments: List[Payment]):
+    return max(payments, default=None, key=attrgetter("pk"))
+
+
+def get_total_captured(payments: List[Payment], fallback_currency: str):
```

Comment:
I see that you're migrating all methods related to payment from Order model.  Don't you think that it would be grat to extract them to separate file? It would be much more easier to find them and IMO everything is better than pumping the utils file 😄

Reply:
@korycins Also thought about it, good idea, will refactor that.